pkg/gadgets/top/ebpf/tracer/tracer.go

```diff
@@ -196,7 +198,7 @@ func getPidMapFromProcFs() (map[uint32][]*types.Process, error) {
 }
 
 func getMemoryUsage(m *ebpf.Map) (uint64, error) {
-	f, err := os.Open(fmt.Sprintf("/proc/self/fdinfo/%d", m.FD()))
+	f, err := os.Open(fmt.Sprintf("%s/proc/self/fdinfo/%d", host.HostRoot, m.FD()))
```

Comment:
Some places use `filepath.Join` and others `fmt.Sprintf`. Can we uniform it? (Given that this code is linux specific, can we only use fmt.Sprintf?)

Reply:
I switched to filepath.Join, I find it more clear for semantics.